src/gatsby-node.js

```diff
@@ -302,6 +304,44 @@ exports.sourceNodes = async (
         );
       }
     };
+    ws.onclose = function() {
+      reconnectingWebSocket(ws, previousDelay);
+    };
+    ws.onerror = function(err) {
+      console.log(err.stack);
+    };
+
+    ws.on('pong', () => {
+      count--;
+    });
+
+    const intervalId = setInterval(() => {
```

Comment:
I believe that you should use setTimeout and schedule new retry after each failed one, with increased delay. Right now I saw it printing endless amount of "we are retrying to connect".

Reply:
And also remember that the delay is in milliseconds.

Reply:
I am also getting the same output

Reply:
Also, because the interval is only cleared on success, every webSocketStart schedules a new interval on failure, adding quite amount of retries.

Reply:
I resolved this issue in below commit :)